### PR TITLE
Community frontend

### PR DESCRIPTION
**Pull request by cuahsimarko**
Customer CZO has requested frontend changes:
- Public groups, remove Overview tab
- Update contact information
- This also contains https://github.com/hydroshare/hydroshare/pull/3596 and the git hashes will collapse cleanly, upon merge to develop

### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. Check public group with at least one public resource that is shared with that group - see that resources are listed right away and no Overview tab appears (recent activity tab)

### CODE REVIEW

**Comment by martinseul** [comment]
Mark, as i understood the request was not to remove the overview tab but only to not have it the first page that opens when clicking into groups. Removing the tab completely changes general HS behaviour.

**Comment by cuahsimarko** [comment]
@martinseul just repeating for everyone else's sake as you and I have just clarified these are different pages. It's not a change to any existing HS behavior/page;  when logged in that view is entirely unchanged. The design for the "new" unauthenticated view has just changed, per engaged customer request.

**Comment by cuahsimarko** [comment]
Groups page is unchanged
image

File: hs_communities/views/communities.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def get_context_data(self, **kwargs):
 
         community = community_from_name_or_id(kwargs['community_id'])
         community_resources = community.public_resources
+        print(len(community_resources))
```
Comment (sblack-usu) [inline hs_communities/views/communities.py]: remove or debug log

File: hs_communities/views/communities.py [hunk 2]
```diff
@@ -104,9 +105,18 @@ class TopicsView(TemplateView):
     """
 
     def get(self, request, *args, **kwargs):
+        u = User.objects.get(pk=self.request.user.id)
+        if u.username not in ['czo_national', 'czo_sierra', 'czo_boulder', 'czo_christina', 'czo_luquillo', 'czo_eel',
```
Comment (sblack-usu) [inline hs_communities/views/communities.py]: Could you throw in a TODO for making this editable in the admin page?

Comment (cuahsimarko) [reply hs_communities/views/communities.py]: https://github.com/hydroshare/hydroshare/issues/3622

**Pull request merged**